### ISSUE TITLE
edge如何打印

### ISSUE THREAD

**Issue by lyc728** [open-mmlab/mmocr#396]
在训练中能否打印出edge的信息，用于后面判断，也可进一步进行分析，如图

<img width="945" alt="微信图片_20210728173131" src="https://user-images.githubusercontent.com/88084847/127299663-ef7d41cb-930a-4d97-933c-cf328b5bf02b.png">

**Comment by gaotongxiao** [comment]
We currently don't have such an utility to print edges for debugging. If necessary, you may consider hacking `mmocr/models/kie/heads/sdmgr_head.py`.

**Comment by lyc728** [comment]
Thanks you reply. I find the edge in mmocr/models/kie/extractors/sdmgr.py.
                edges=F.softmax(edge_preds, -1))
        ]
I only want to get the result of edge and print it.

**Comment by lyc728** [comment]
> We currently don't have such an utility to print edges for debugging. If necessary, you may consider hacking `mmocr/models/kie/heads/sdmgr_head.py`.

Thanks you reply. I find the edge in mmocr/models/kie/extractors/sdmgr.py.
edges=F.softmax(edge_preds, -1))
]
I only want to get the result of edge and print it.

**Comment by gaotongxiao** [comment]
Looks like `print(edge_preds)` is what you need.

**Comment by lyc728** [comment]
> Looks like `print(edge_preds)` is what you need.

But print(edge_preds) where should this code go? 

                img_metas=img_metas,
                nodes=F.softmax(node_preds, -1),
                edges=F.softmax(edge_preds, -1))
        ]
    print(edge_preds)
It's not show the result

**Comment by gaotongxiao** [comment]
Because you have added it **after the return statement**...

**Comment by lyc728** [comment]
> Because you have added it **after the return statement**...

Hello, can you add me qq for contact? Thank you very much
1335817464

**Comment by lyc728** [comment]
> Because you have added it **after the return statement**...

How do I add code to the right place, please help me!
